frontend/src/components/Header/Header.tsx

```diff
@@ -50,21 +74,67 @@ function Header({ organisation }: HeaderProps) {
-                        component="img"
-                        src={THEME_TO_IMAGE[organisation]}
-                        alt="Organisation logo"
-                    />
-                </Link>
-                {showDateAndTime && (
-                    <Typography variant="h4" component="p">
-                        {date}
-                    </Typography>
-                )}
+                <Tooltip followCursor={true} title="Return to home screen">
+                    <Link to="/">
+                        <Box sx={{ alignItems: "center", display: "flex" }}>
+                            <Box
+                                height={{ xs: "2rem", md: "2.5rem" }}
+                                component="img"
+                                src={THEME_TO_IMAGE[props.organisation]}
+                                alt="Organisation logo"
+                            />
+                            {props.isAuthenticated ?
+                                <IconButton {...buttonProps}>
+                                    <HomeIcon fontSize="large" />
+                                </IconButton> : null
+                            }
+                        </Box>
+                    </Link>
+                </Tooltip>
+
+                <Box
+                    sx={{ alignItems: "center", display: "flex" }}
+                    id="rightDiv">
+                    <Tooltip title="Open documentation">
+                        <IconButton
+                            {...buttonProps}
+                            onClick={() => setOpenHelpModal(true)}
+                        >
+                            <HelpIcon fontSize="large" />
+                        </IconButton>
+                    </Tooltip>
+                    {showDateAndTime && (
+                        <Typography variant="h4" component="p">
+                            {date}
+                        </Typography>
+                    )}
+                    {props.isAuthenticated ?
+                        <Tooltip title="Log out">
+                            <IconButton
+                                {...buttonProps}
+                                onClick={() => setOpenLogoutModal(true)}
+                            >
+                                <LogoutIcon fontSize="large" />
+                            </IconButton>
+                        </Tooltip> : null
+                    }
+                </Box>
             </Toolbar>
-        </AppBar>
+            <HeaderDialog
+                openLogoutModal={openLogoutModal}
+                setOpenLogoutModal={setOpenLogoutModal}
+                callback={signOut}
+                dialogTitle="Log out?"
+                dialogContent=""
+            />
+            <HeaderDialog
+                openLogoutModal={openHelpModal}
+                setOpenLogoutModal={setOpenHelpModal}
+                callback={() => window.open('https:/docs.utn.se/ordsys/frontend', '_blank')}
```

Comment:
I figured that the frontend user interface is what most people would want to read so by linking directly to it would save people confusion, but I suppose that's up for debate

Reply:
I agree with you. I think the proper solution would be to rename the title in the documentation repository